crates/models/src/authorizations.rs

```diff
@@ -105,6 +108,53 @@ pub struct UserCollectionAuthorization {
     pub retry_millis: u64,
 }
 
+#[derive(
+    Debug, serde::Serialize, serde::Deserialize, schemars::JsonSchema, validator::Validate,
+)]
+#[serde(rename_all = "camelCase")]
+pub struct UserPrefixAuthorizationRequest {
+    /// # Prefix to be authorized.
+    #[validate]
+    pub prefix: crate::Prefix,
+    /// # Name of the data-plane to be authorized.
+    #[validate]
+    pub data_plane: crate::Name,
+    /// # Requested capability level of the authorization.
+    #[serde(default = "capability_read")]
+    pub capability: crate::Capability,
+    /// # Unix timestamp, in seconds, at which the operation started.
+    /// This timestamp lower-bounds the time of an authorization
+    /// snapshot required to definitively reject an authorization.
+    ///
+    /// Snapshots taken prior to this time point that reject the request
+    /// will return a Response asking for the operation to be retried.
+    pub started_unix: u64,
+}
+
+#[derive(Debug, Default, serde::Serialize, serde::Deserialize, schemars::JsonSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct UserPrefixAuthorization {
```

Comment:
nit: <code>UserPrefixAuthorization<b>Response</b></code> to match other `Request`/`Response` structs? Or maybe it's really just `DekafAuthResponse` down below that should be renamed to `DekafAuth` 🤔 

I very minorly like `Response` better as it's clear that this is the response from a network call, but 6 of one half-dozen of the other

Reply:
all of the others don't use `Response`